nobl9/release_channel_schema.go

```diff
@@ -15,7 +16,11 @@ func schemaReleaseChannel() *schema.Schema {
 	return &schema.Schema{
 		Type:        schema.TypeString,
 		Optional:    true,
+		Computed:    true,
 		Description: "Release channel of the created datasource [stable/beta]",
+		ValidateDiagFunc: validation.ToDiagFunc(
+			validation.StringInSlice([]string{"stable", "beta"}, false),
```

Comment:
There's an enum for these in SDK:
```suggestion
			validation.StringInSlice([]string{v1alpha.ReleaseChannelStable.String(), v1alpha.ReleaseChannelBeta.String()}, false),
```